utility/src/main/java/org/oppia/util/caching/AssetRepository.kt

```diff
@@ -34,19 +37,22 @@ class AssetRepository @Inject constructor(private val context: Context, private
     }
   }
 
-  /** Ensures the contents corresponding to the specified asset are available for quick retrieval. */
+  /** Ensures the contents corresponding to the specified asset are available for
+   * quick retrieval.
+   */
   fun primeTextFileFromLocalAssets(assetName: String) {
     repositoryLock.withLock {
       if (assetName !in textFileAssets) {
         logger.d("AssetRepo", "Caching local text asset: $assetName")
-        textFileAssets[assetName] = context.assets.open(assetName).bufferedReader().use { it.readText() }
+        textFileAssets[assetName] = context.assets.open(assetName).bufferedReader().use {
+          it.readText() }
```

Comment:
Similarly, this wrapping doesn't seem correct. The ``}`` should be on its own line. Also, this wrapping doesn't seem to follow the Kotlin style guide of 'breaking at the higher syntactical element': https://developer.android.com/kotlin/style-guide#where_to_break. Following that, we would break at the ``=``, then at the ``.``s, and finally at ``{`` if there wasn't enough room.

Reply:
+1
Having `}` on separate line makes code more readable.

Reply:
+1